packages/hdwallet-gridplus/src/gridplus.ts

```diff
@@ -296,119 +294,77 @@ export class GridPlusHDWallet
   readonly _supportsSolana = true;
   readonly _supportsThorchain = true;
 
-  readonly _isGridPlus = true;
```

Comment:
blocking: bring me back.
This breaks skip autodiscovery checks and makes gridplus virtually unusable in web i.e for my accounts set, ~100 reqs and stuck for a good 2-3mn

<img width="50" height="815" alt="Image" src="https://github.com/user-attachments/assets/e8095005-96ce-473c-9751-6a368ebe3edb" />